network-area-diagram/src/main/java/com/powsybl/nad/build/iidm/NetworkGraphBuilder.java

```diff
@@ -99,16 +99,16 @@ private void visitThreeWindingsTransformer(VoltageLevel vl, ThreeWindingsTransfo
         ThreeWtNode tn = new ThreeWtNode(idProvider.createId(thwt), thwt.getId(), thwt.getNameOrId());
         graph.addNode(tn);
 
-        ThreeWindingsTransformer.Side side;
+        ThreeSides side;
         if (thwt.getLeg1().getTerminal().getVoltageLevel() == vl) {
-            side = ThreeWindingsTransformer.Side.ONE;
+            side = ThreeSides.ONE;
         } else if (thwt.getLeg2().getTerminal().getVoltageLevel() == vl) {
-            side = ThreeWindingsTransformer.Side.TWO;
+            side = ThreeSides.TWO;
         } else {
-            side = ThreeWindingsTransformer.Side.THREE;
+            side = ThreeSides.THREE;
         }
```

Comment:
We do not have full coverage here, and I realize this code needs some cleaning... Who did write this?? :sweat:  Could you try doing this with a stream on the `ThreeSides.values()`?